### PR TITLE
Adding Date Modified on Articles Manager (new feature)

### PR DESCRIPTION
**Pull request by joomlabeat**
Adding Date Modified sorting for Articles Manager in backend

Pull Request for Issue #15610

### Summary of Changes
4 files modified 
1. Including Date Modified field in the model 
  administrator/components/com_content/models/articles.php

2. Adding the option in the ordering list form field with language constant.
 administrator/components/com_content/models/forms/filter_articles.xml

3.  Modifying the tmpl file, to consider the new date ordering label on the table's headings.
  administrator/components/com_content/views/articles/tmpl/default.php

4. Adding en-GB translations for the new constants
   administrator/language/en-GB/en-GB.com_content.ini
  Totally 3 new constants:
COM_CONTENT_HEADING_DATE_MODIFIED="Date Modified"
COM_CONTENT_MODIFIED_ASC="Date Modified ascending"
COM_CONTENT_MODIFIED_DESC="Date Modified descending"

### Testing Instructions
After applying the patch, go in articles manager in backend and through the Sorting List select the Date Modified ASC/DESC options to sort articles by date modified.

### Expected result
The articles should be ordered by this date option.

### Documentation Changes Required
Mention that articles manager now supports ordering the articles by Date Modified using the Sorting List field.

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on bb17ce39ad27c18c30cd0dd701453fc18fd071d9<br /><br />As this is implemented in the same way as the other date modifiers that are not displayed in the columns by default this is fine by me.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15825">issues.joomla.org/tracker/joomla-cms/15825</a>.</sub>

File: administrator/components/com_content/models/forms/filter_articles.xml [hunk 1]
```diff
@@ -110,6 +110,8 @@
 			<option value="language DESC">JGRID_HEADING_LANGUAGE_DESC</option>
 			<option value="a.created ASC">JDATE_ASC</option>
 			<option value="a.created DESC">JDATE_DESC</option>
```
Comment (Quy) [inline administrator/components/com_content/models/forms/filter_articles.xml]: Fix tab alignment.

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 2]
```diff
@@ -161,6 +162,8 @@ COM_CONTENT_PAGEBREAK_DOC_TITLE="Page Break"
 COM_CONTENT_PAGEBREAK_INSERT_BUTTON="Insert Page Break"
 COM_CONTENT_PAGEBREAK_TITLE="Page Title:"
 COM_CONTENT_PAGEBREAK_TOC="Table of Contents Alias:"
+COM_CONTENT_MODIFIED_ASC="Date Modified ascending"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_content.ini]: Alphabetize/move strings.

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on bb17ce39ad27c18c30cd0dd701453fc18fd071d9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15825">issues.joomla.org/tracker/joomla-cms/15825</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**